using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
 Deleting your mobile property in Experience Platform Launch does not automatically delete your Campaign Standard mobile app.
 {% endhint %}
 
+### Handling deeplinks included in Campaign In-App messages
+
+A deeplink can be added to in-app messages delivered from Adobe Campaign. The link can be a website URL such as https://www.adobe.com or an app link which can be used to direct the user to a specific area of your app.
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Handling in-app message deeplinks on Android
+
+Website URL's are handled without any further action by the app developer. If an in-app message is clicked through and contains a valid URL, the device's default web browser will redirect to the URL contained in the in-app notification payload's `url` key (for fullscreen and alert messages) or `adb_deeplink` key (for local and push notifications). App links present in the notification payload can be handled by setting up URL schemes in the app. For more information about setting URL schemes for Android, see [Create Deep Links to App Content](https://developer.android.com/training/app-links/deep-linking).
```

Comment:
- In the 1st sentence, please remove the "'s" in URL's --> "URLs" and replace "without any further action..." with "without any additional action..."
- The 2nd sentence is really long, so I suggest breaking it up as follows:
"If an in-app message is clicked through and contains a valid URL, the device's default web browser redirects the user to the URL in the in-app notification payload's `url` in one of the following ways:
    * key (for fullscreen and alert messages)
    * `adb_deeplink` key (for local and push notifications)"

- The 3rd sentence is also pretty long, so I suggest rewording it like this:
"To handle app links in the notification payload, you need to set up URL schemes in the app. For more information about setting URL schemes for Android, see [Create Deep Links to App Content](https://developer.android.com/training/app-links/deep-linking)."

Reply:
I've reworded the second bullet point's suggestion since the`url` key is part of the fullscreen and alert message payload.